namespace-provisioner/use-case4.hbs.md

```diff
@@ -5,66 +5,42 @@ work with private Git repositories.
 
 ## Disable Grype install
 
-Namespace Provisioner creates Grype scanner install as one of the [default resources](reference.md#default-resources). If you want to use another scanner for namespaces instead of Grype, disable
-Grype scanner as follows:
+If you prefer to utilize a different scanner for namespaces instead of Grype, you have the option to deactivate the installation of the default Grype scanner, which is automatically created by the Namespace Provisioner. To do so, you can follow these steps:
 
-1. Create an overlay secret as follows which removes the Grype scanner and the secret that is automatically created by Namespace Provisioner.
+### Disable for all namespaces
```

Comment:
You might get `woke` error. Would you want to call it `Deactivate`?

Reply:
Fixed woke findings